casper/src/test/scala/coop/rchain/casper/api/BondedStatusAPITest.scala

```diff
@@ -39,7 +41,7 @@ class BondedStatusAPITest
       blockApi <- createBlockApi(node)
       res <- blockApi
               .bondStatus(ByteString.copyFrom(publicKey.bytes), block.some)
-              .map(_.right.value)
+              .map(_.value)
```

Comment:
Is this change somehow related to this PR?

Reply:
Oh, I see now this is an improvement in the new version of ScataTest to extract value from Either with test failed error.